src/routes/NonAuthLayout/NonAuthLayout.js

```diff
@@ -49,18 +49,21 @@ export default class NonAuthLayout extends React.Component {
       } />}
 
       {!HOLOCHAIN_ACTIVE && <Route path='/login' component={() =>
-        <Login {...this.props} styleName='form' />
+        <div styleName='signupRow'>
```

Comment:
i think you want this div inside the <Route> because you dont want it to display unless its on the <Login> route, right?

Reply:
done